src/index.js

```diff
@@ -1,15 +1,25 @@
 import { Piscina } from "piscina";
 import supportsColor from "supports-color";
 import { MessageChannel } from "worker_threads";
+import { shouldInstrument } from "@jest/transform";
+import { fileURLToPath } from "url";
 
 /** @typedef {import("@jest/test-result").Test} Test */
 
 export default class LightRunner {
   #config;
+  #testContext;
   #piscina;
 
-  constructor(config) {
+  constructor(config, testContext) {
     this.#config = config;
+    this.#testContext = testContext;
+
+    const { collectCoverage, coverageProvider } = this.#config;
+
+    if (collectCoverage && coverageProvider !== "v8") {
+      throw new Error("Coverage needs v8 coverage provider");
```

Comment:
it might be better to just log a warning instead of throwing, i.e. in the case where

Reply:
I'd prefer either throw an error, or ignore settings, just use `v8`.

Reply:
People only care where is my coverage report? Who care where is it from.

Reply:
It's ok with the error. If someone uses a different provider and decides to use `jest-light-runner`, it's better to notify them loudly.